_code/InSituTree/R/summarizeInSituTree.R

```diff
@@ -0,0 +1,56 @@
+#' Summarize the annotations for runInSituTree() function
+#'
+#' @param nestList Nested insitutype objects returned from runInSituType()
+#' @param annotationLevel Used for recursive looping.  Do not adjust.
+#'
+#' @return Dataframe of celltype annotation and posterior probabilities
```

Comment:
`data.frame` is a specific data class, might be good to use the exact words in man.

Reply:
Changed to `data.frame`